utils/arrayBuffers.ts

```diff
@@ -0,0 +1,33 @@
+export const toUint8 = (buf: Buffer): Uint8Array => {
+  const uin = new Uint8Array(buf.length);
+  return uin.map((a, index, arr) => (arr[index] = buf[index]));
```

Comment:
I'm not sure we really need this method since you can convert it in one line. If we keep it, could you rather change it to this?

```suggestion
return new Uint8Array(buf);
```